tests/system/test_system_pandasdf.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import spanner_v1_mod
+from google.auth.credentials import AnonymousCredentials
+import pytest
+
+# for referrence
+TABLE_NAME = 'testTable'
+COLUMNS = ['id', 'name']
+VALUES = [[1, 'Alice'], [2, 'Bob']]
+
+
+TABLE_NAME_1 = 'Functional_Alltypes'
```

Comment:
This looks like the name from the BigQuery Ibis tests. Please use the existing `all_types` table from https://github.com/googleapis/python-spanner/blob/e990ff70342e7c2e27059e82c8d74cce39eb85d0/tests/_fixtures.py#L31 if possible.

Reply:
[Optional] In fact, we may be able to import test data from here:

https://github.com/googleapis/python-spanner/blob/e990ff70342e7c2e27059e82c8d74cce39eb85d0/tests/system/test_system.py#L1096

For example, the DB-API tests import some useful test fixtures.

https://github.com/googleapis/python-spanner/blob/e990ff70342e7c2e27059e82c8d74cce39eb85d0/tests/system/test_system_dbapi.py#L32-L39